docs/en/stack/data-frames/limitations.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[dataframe-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-wizard-limitations]]
+=== {dataframe-cap} wizard limitation
+
+If you have fields that contain arrays, you cannot select these fields in the source 
+index table in the {kib} {dataframe} wizard. You can, however, use those fields 
+when you group and aggregate your data in the wizard.
+
+[float]
+[[df-ccs-limitations]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
+
+[float]
+[[df-kibana-limitations]]
+=== Displaying {dataframe-transforms} in {kib} limitation
+
+The {kib} *Machine Learning* > *Data Frames* page lists up to 100 {dataframe-transforms}.
+You can work-around this limitation by calling the 
+{ref}/get-data-frame-transform.html[GET {dataframe-transforms} API] 
+with the `size` parameter.
+
+[float]
+[[df-dateformat-limitations]]
+=== Date histogram limitation
+
+If you use a {ref}/search-aggregations-bucket-datehistogram-aggregation.html[date 
+histogram] in the `group_by` object in the create or preview {dataframe-transform} 
+APIs, the defined interval and time format must have the same time fidelity, 
+otherwise it might cause issues in the {dataframe}.
+
+For example, if you set the `calendar_interval` of the date histogram to one minute 
+(`1m`), then make sure that the `format` is `yyyy-MM-dd HH:mm` instead of 
+`yyyy-MM-dd HH:00`.
+
+[float]
+[[df-aggresponse-limitations]]
+=== Strings in aggregation responses
```

Comment:
This particular issue has been fixed.. however there is a window for some similar silent failures... perhaps change to something along the lines of  ..


`Aggregation responses may be incompatible with destination index mappings`

`Data frames use composite aggregations in order to transform data. In some cases, composite aggregations may return responses which are not compatible with the mappings set for the destination index. For example "NaN", "Infinity" or possibly a numeric overflow. Where possible, a null response has been substituted. Please check elasticsearch logs if you think this may have occurred. As a workaround, using the API, you may define custom destination index mappings prior to starting the data frame transform.`